src/main/java/tour/Main.java

```diff
@@ -42,14 +43,17 @@ public static void main(String[] args) throws Exception {
         System.out.println("Running the Accumulo tour. Having fun yet?");
 
         Path tempDir = Files.createTempDirectory(Paths.get("target"), "mac");
-        MiniAccumuloCluster mac = new MiniAccumuloCluster(tempDir.toFile(), "tourguide");
+        MiniAccumuloCluster mac = new MiniAccumuloCluster(tempDir.toFile(), "tourpass");
 
         mac.start();
-        exercise(mac);
+
+	try (AccumuloClient client = mac.createAccumuloClient("root", new PasswordToken("tourpass"))) {
```

Comment:
nit; replace tabs with spaces.
```suggestion
	try (AccumuloClient client = mac.createAccumuloClient("root", new PasswordToken("tourpass"))) {
```